app/web/src/features/communities/NewGuidePage.tsx

```diff
@@ -6,6 +7,7 @@ import NewGuideForm from "./NewGuideForm";
 export default function NewGuidePage() {
   return (
     <>
+      <HtmlMeta title={"Create a new guide"} />
```

Comment:
Just for the record, this page is basically a placeholder which is why it isn't using constants (yet)